### PR TITLE
Fix small documentation issues pointed out in #238, #245, and #247.

### PR DESCRIPTION
**Pull request by jdmgoogle**
Fix small documentation issues pointed out in #238, #245, and #247.

There needs to be another round of fixes since there are still some inconsistencies, but I'd like to do that separately from these changes.

### CODE REVIEW

File: docs/yaml/elements/election.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: Election
 tags:
 - _name: Date
-  description: Specifies when the election is being held. The Date is considered to
+  description: Specifies when the election is being held. The `Date` is considered to
     be in the timezone local to the state holding the election.
   error: If the field is not present or invalid, the implementation is required to
-    ignore the election element containing it.
+    ignore the `Election` element containing it.
```
Comment (cjerdonek) [inline docs/yaml/elements/election.yaml]: For this one, you can delete the `error` key-value entirely.  What you have is what the default message will be.  (Going backwards on this and automatically performing this "DRY-ing" up was one of the things I didn't have time to get to.)

File: docs/yaml/elements/election.yaml [hunk 2]
```diff
@@ -13,8 +13,9 @@ tags:
   error_then: =must-ignore
   type: InternationalizedText
 - _name: StateId
-  description: Specifies a link to the state element where the election is being held.
-  error_then: =must-ignore
+  description: Specifies a link to the `State` element where the election is being held.
+  error: If the field is not present or invalid, the implementation is required to
+    ignore the `Election` element containing it.
```
Comment (cjerdonek) [inline docs/yaml/elements/election.yaml]: Same here, you can delete the `error` key-value entirely.

File: docs/yaml/elements/source.yaml [hunk 3]
```diff
@@ -2,22 +2,19 @@ name: Source
 tags:
 - _name: Name
   description: Specifies the name of the organization that is providing the information.
-  error: If the field is invalid, the implementation is required to ignore the source
+  error: If the field is invalid, the implementation is required to ignore the `Source`
```
Comment (cjerdonek) [inline docs/yaml/elements/source.yaml]: Ditto

File: docs/yaml/elements/source.yaml [hunk 4]
```diff
@@ -2,22 +2,19 @@ name: Source
...
   required: true
   type: xs:string
 - _name: VipId
-  description: Specifies the ID of the organization as assigned by VIP. This ID is
-    not an attribute so as not to interfere with organizations' own numbering conventions
-    (since id attributes must be unique across the entire file).
-  error: If the field is invalid, the implementation is required to ignore the source
+  description: Specifies the ID of the organization. VIP uses FIPS_ codes for this ID.
+  error: If the field is invalid, the implementation is required to ignore the `Source`
     element containing it.
```
Comment (cjerdonek) [inline docs/yaml/elements/source.yaml]: ditto

**Comment by cjerdonek** [comment]
Looks good to me other than what I suggested above and here.

One suggestion on PR's involving the tables that I included in the updated CONTRIBUTING docs:

You will also need to commit the updated table files (after running `update_tables), so it's also good to include those files in the PR.  That way others can see what the result of the YAML changes are (e.g. the default error messages, etc).

**Comment by jungshadow** [comment]
LGTM with @cjerdonek's edits.

**Comment by jdmgoogle** [comment]
@cjerdonek @jungshadow Updated the tables and added *.pyc to the .gitignore. I also updated the CONTRIBUING.md file based on my experience.

PTAL

**Comment by cjerdonek** [comment]
LGTM!

**Comment by jungshadow** [comment]
LGTM

**Pull request merged**